tests/Cronos.Tests/CronExpressionFacts.cs

```diff
@@ -18,10 +18,12 @@ public class CronExpressionFacts
         private static readonly string EasternTimeZoneId = IsUnix ? "America/New_York" : "Eastern Standard Time";
         private static readonly string JordanTimeZoneId = IsUnix ? "Asia/Amman" : "Jordan Standard Time";
         private static readonly string LordHoweTimeZoneId = IsUnix ? "Australia/Lord_Howe" : "Lord Howe Standard Time";
+        private static readonly string BrusselsTimeZoneId = IsUnix ? "Europe/Brussel" : "W. Europe Standard Time";
```

Comment:
Looks like you forgot 's' character at the end of `BrusselsTimeZoneId` for Unix.